README.md

```diff
@@ -89,7 +89,10 @@ limited by the capacity of a single physical machine, lest we sacrifice the
 property of complete auditability.  The Lightning Network [\[7\]][7] can help
 scale Bitcoin transaction volume by leaving some transactions off the ledger
 completely and is well suited for micropayments and privacy preserving payment
-rails, but may not be suitable for more generalized scaling needs.
+rails, and it is suitable for more generalized scaling needs.
+<!---
+state-channels allow for generalized smart contracts in the lightning network
+--->
```

Comment:
I don't understand your comment. What is about tokens? why does impermanence matter? why do you say "asymmetric token transfer" and what does it mean?

Channels have generalized smart contracts the same as on-chain contracts.
Domain registration contracts are less efficient off-chain, but most other things are more efficient off-chain.

The lightning network is not constrained to a single ledger. You can have smart contracts that span multiple blockchains.